client/a8c-for-agencies/components/sidebar/header/profile-dropdown.tsx

```diff
@@ -32,13 +33,22 @@ const DropdownMenu = ( { isExpanded, setMenuExpanded }: DropdownMenuProps ) => {
 		dispatch( redirectToLogout() );
 	}, [ dispatch ] );
 
+	const isClient = isClientView();
+
 	return (
 		<ul className="a4a-sidebar__profile-dropdown-menu" hidden={ ! isExpanded }>
-			<li className="a4a-sidebar__profile-dropdown-menu-item">
-				<Button borderless onClick={ onGetHelp }>
-					{ translate( 'Contact support' ) }
-				</Button>
-			</li>
+			{
+				// Show the "Contact support" button if the user is not a client
+				! isClient && (
+					<>
```

Comment:
is the extra <> needed here?

Reply:
Not required. Removed it [here](https://github.com/Automattic/wp-calypso/pull/91934/commits/cff47a1736ab1848a45c5bcfc200b09d959feffe)